admin_scripts/reposync/reposync.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
Add trap, XCAT_TOOLS_TOP_PID, BASE (copy lines 2-5 in `mk_new_node.sh`)

Reply:
Looks like I'd need to copy the loop that loads in the libs too right? Didn't look like the trap/XCAT_TOOLS_TOP_PID were very useful without at least importing the logging lib.

Reply:
These changes have been added to the script.